library/platform_util.c

```diff
@@ -79,6 +79,71 @@ void mbedtls_platform_zeroize( void *buf, size_t len )
 }
 #endif /* MBEDTLS_PLATFORM_ZEROIZE_ALT */
 
+void mbedtls_platform_memset( void *ptr, int value, size_t num )
+{
+    /* Randomize start offset. */
+    size_t startOffset = mbedtls_random_in_range( num );
+    /* Randomize data */
+    size_t data = mbedtls_random_in_range( 0xff );
+
+    /* Perform a pair of memset operations from random locations with
+     * random data */
+    memset( ( void * ) ( ptr + startOffset ), value, ( num - startOffset ) );
```

Comment:
ARMC5 compiler gives "_platform_util.c", line 91: Error:  #852: expression must be a pointer to a complete object type"_. Would you please add type cast like already done in other functions?

Reply:
Done

Reply:
I verified this is done, and the CI is happy with the fix as well, so I'm marking the conversation as resolved in @artokin's absence.